### PR TITLE
Add base64 encoded token response form authenticate to changelog

### PR DESCRIPTION
**Pull request by liavyona**
### What does this PR do?
Add base64 encoded token response form authenticate to changelog

[slack](https://conjurhq.slack.com/archives/CGTRB6G8G/p1598892584024500)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,14 +6,17 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
-  identity token. See [design](design/authenticators/authn_gce/authn_gce_solution_design.md) 
+- Hosts can authenticate from Google Compute Engines (GCE) using a GCE instance
+  identity token. See [design](design/authenticators/authn_gce/authn_gce_solution_design.md)
   for details ([cyberark/conjur#1711](https://github.com/cyberark/conjur/issues/1711)).
 - Add `/whoami` API endpoint for improved supportability and debugging for access
   tokens and client IP address determination. [cyberark/conjur#1697](https://github.com/cyberark/conjur/issues/1697)
 - `TRUSTED_PROXIES` is validated at Conjur startup to ensure that it contains
   valid IP addresses and/or address ranges in CIDR notation.
   [cyberark/conjur#1727](https://github.com/cyberark/conjur/issues/1727)
+- `/authenticate` endpoint now returns the access token as a base64 encoded
```
Comment (orenbm) [inline CHANGELOG.md:17]: "The `/authenticate` endpoint now returns a base64 encoded access token if the `Accept-Encoding` request header includes `base64`"

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,14 +6,17 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+  string if the `Accept-Encoding` request header includes `base64`.
+  [cyberark/conjur#151](https://github.com/cyberark/conjur/issues/151)
```
Comment (orenbm) [inline CHANGELOG.md:19]:
```suggestion
- "The `/authenticate` endpoint now returns a base64 encoded access token
   if the `Accept-Encoding` request header includes `base64`".
   [cyberark/conjur#151](https://github.com/cyberark/conjur/issues/151)
```

**Review by orenbm — CHANGES_REQUESTED** [review]
added a suggestion

File: CHANGELOG.md (hunk 2 above)
Comment (uCatu) [reply CHANGELOG.md:19]: Why the quotes? and the space before link

Comment (uCatu) [inline CHANGELOG.md:19]:
```suggestion
- The `/authenticate` endpoint now returns a text/plain base64 encoded access token
  if the `Accept-Encoding` request header includes `base64`.
  [cyberark/conjur#151](https://github.com/cyberark/conjur/issues/151)
```

**Review by uCatu — CHANGES_REQUESTED** [review]
WIP

**Review by orenbm — APPROVED** [review]

**Review by uCatu — APPROVED** [review]
LGTM

**Pull request merged**